src/app/components/Drbb/DrbbItem.jsx

```diff
@@ -2,6 +2,8 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Link } from 'react-router';
 
+import DownloadIcon from '../../../client/icons/Download';
```

Comment:
Check this in Safari.  With any luck, it'll just work, but I've had issues with svgs not loading in Safari before.

Reply:
edit: oh I see below that it's saved as a .jsx, that should work.  Checking in Safari is a good thing to do anyway haha